components/http-accept.js

```diff
@@ -19,35 +19,43 @@ exports.getComponent = promiseComponent({
             description: "HTTP 202 Accepted request/response pair {req, res}",
             datatype: 'object'
         }
-    },
+    }, promiseOutput.outPorts),
     inPorts: {
         limit: {
             description: "Request Content-Length limit",
             datatype: 'int',
-            ondata: basenode.assign('limit')
+            ondata: function(limit) {
+                this.nodeInstance.limit = limit;
```

Comment:
Until the discussion this morning, I would have been concerned this should be this.nodeInstance.rpf.  Will this will now be assigned to the noflo facade?